app/src/main/java/com/teamagam/gimelgimel/app/message/viewModel/adapter/MessagesRecyclerViewAdapter.java

```diff
@@ -67,14 +74,44 @@ public synchronized void select(String messageId) {
         notifyDataSetChanged();
     }
 
+    private synchronized void selectWithEffect(MessageViewHolder viewHolder) {
+        RelativeLayout container = viewHolder.container;
+
+        ObjectAnimator colorFade = ObjectAnimator.ofObject(
+                container,
+                "backgroundColor",
+                new ArgbEvaluator(), 0x00000000, 0x603F51B5);
```

Comment:
Use colors.xml instead of magic ints.

You can convert them to color ints with this:
ContextCompat.getColor(context, R.color.red)

you can get a context from the viewholder:
viewHolder.itemView.getContext()
 
(it works with any view , so you can also use container.getContext())